integrate.js

```diff
@@ -65,6 +65,9 @@ WebApp._onPageReady = function()
 {
     // Connect handler for signal ActionActivated
     Nuvola.actions.connect("ActionActivated", this);
+    document.querySelector(".centerblock-wrapper").setAttribute('style', 'width: 100%');
+    var sidebar = document.querySelector(".sidebar__placeholder .sidebar");
+    sidebar.parentNode.removeChild(sidebar);
```

Comment:
The same here. The `sidebar` variable may be `null` and throw an exception. You can use a single try-catch block.